redirect/routes.php

```diff
@@ -329,6 +331,17 @@ public function getTargetOfSub($host) {
         if (isset($this->synonyms[$redirectUrl])) {
             $redirectUrl = $this->synonyms[$redirectUrl];
         }
+        //moodle support; example: mgad.tum.sexy
+        elseif (strlen($redirectUrl) > 1 && substr($redirectUrl, 0, 1) === 'm' && !isset($this->routes[$redirectUrl])){
+            $tmp = substr($redirectUrl, 1);
+            if (isset($this->synonyms[$tmp])) {
```

Comment:
Would be nice to know in a comment, what your intentions are here... ("Allow for synonyms as well in moodle redirects")